src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -3892,24 +3904,20 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         return std::max(CharacterDimensions().Width * 2.0 / 3.0, GetPadding().Left);
     }
 
-    void TermControl::OpenQuickFixMenu()
+    bool TermControl::OpenQuickFixMenu()
     {
         if (Feature_QuickFix::IsEnabled() && _core.QuickFixesAvailable())

```

Comment:
nit: prefer `if constexpr` for feature checks, even if it means introducing another level of nexting